tests/AssertTraitTest.php

```diff
@@ -59,27 +49,27 @@ public function testAssertJsonSchemaFailMessage()
     /**
      * Tests if referenced schemas are loaded automatically.
      */
-    public function testAssertJsonSchemaWithRefs()
+    public function testAssertJsonMatchesSchemaWithRefs()
     {
         $content = json_decode('{"code":123, "message":"Nothing works."}');
 
-        AssertTraitImpl::assertJsonMatchesSchema(self::getSchema('error.schema.json'), $content);
+        AssertTraitImpl::assertJsonMatchesSchema(Utils::getSchemaPath('error.schema.json'), $content);
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Avoid using static access to class 'EnricoStahn\JsonAssert\Tests\AssertTraitImpl' in method 'testAssertJsonMatchesSchemaWithRefs'.](https://www.codacy.com/app/estahn/phpunit-json-assertions/file/2256991340/issues/source?bid=3181605&fileBranchId=3182397#l56)